neoexchange/core/views.py

```diff
@@ -3942,15 +3956,140 @@ def import_alcdef(file, meta_list, lc_list):
             elif 'STARTMETADATA' in line:
                 met_dat = True
             elif 'ENDMETADATA' in line:
+                metadata['alcdef_url'] = reverse('display_textfile', kwargs={'pk': alcdef.id})
                 met_dat = False
 
     return meta_list, lc_list
 
 
+def import_period_scan(file, scan_list):
+    """Pull Period data from Period Scan files."""
+
+    scan_data = {'period': [], 'rms': [], 'chi2': []}
+    ps_file = file.open()
```

Comment:
so, it's not closed because it's never really opened. 
The dataproduct is implicitly opened when it's called. It can't be opened with `open()`.

I originally included the `.open()` because I was also testing this on non-data products (back before those existed.)
I've removed the `.open()`s all together, and just rely on the implicit open that occurs anytime the DP is referenced.

I'm not sure if it makes sense to include `.close()` anywhere at this point...